README.md

```diff
@@ -1,2 +1,10 @@
-# babel-preset-enact-app
-Babel preset used by Enact
+# babel-preset-enact
```

Comment:
Can we update README with the detailed information? If it cannot be done now, let's create another ticket.

Reply:
OK, I will create another ticket.

Reply:
WRP-7819